Kconfig

```diff
@@ -72,6 +72,15 @@ config AWS_IOT_MQTT_MAX_RECONNECT_WAIT_INTERVAL
         Maximum delay between reconnection attempts. If the exponentially increased delay
         interval reaches this value, the client will stop automatically attempting to reconnect.
 
+config AWS_IOT_USE_HARDWARE_SECURE_ELEMENT
+    bool "Allow hardware secure element for the TLS"
+    depends on ATCA_MBEDTLS_ECDSA
+    help
+        If you have the Microchip cryptolib (from https://github.com/espressif/esp-cryptoauthlib) for using a 
```

Comment:
Suggest bit rephrasing: `If you have the fork of the Microchip cryptoauthlib`...